grass7/imagery/i.landsat/i.landsat.download/i.landsat.download.py

```diff
@@ -81,26 +90,30 @@
 #% options: RT, T1, T2
 #% guisection: Filter
 #%end
+
 #%option
 #% key: sort
 #% description: Sort by values in given order
 #% multiple: yes
-#% options: acquisitionDate,cloudCover
-#% answer: cloudCover,acquisitionDate
+#% options: publishDate,cloudCover
```

Comment:
I would suggest to keep it as `acquisitionDate`, it is more understandable by the user (we can internally change the name later)